CONTRIBUTING.md

```diff
@@ -66,22 +39,118 @@ If you've found a bug that is not on the board, [follow these steps](#found-a-bu
 
 > _Updating to the latest releases is recommended_.
 
-### Getting Started
+### Forking The Project
 
-Note: If this is your first time working with a node-gyp dependent module,
-please follow the [node-gyp installation
-guide](https://github.com/nodejs/node-gyp#installation) to ensure a working npm
-build.
+#### Setting Up Your System
 
-The easiest way to get started is to clone the repository:
+1. Install [Git](https://git-scm.com/) or your favourite Git client.
+2. (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-ssh-keys/) for GitHub.
```

Comment:
To prevent redirect, URL should be `https://help.github.com/articles/generating-an-ssh-key/`